CONTRIBUTING.md

```diff
@@ -45,8 +45,19 @@ The usual contributing steps are:
 
 ### Contributing to the documentation.
 
-Gatsby, unsurprisingly uses Gatsby for it's documentation website. Here are the
-steps to add/modify documentation and preview it.
+Gatsby, unsurprisingly, uses Gatsby for it's documentation website.
+
+If you want to add/modify some documentation markdown to make a change in the
+website, go to the
```

Comment:
It sounds more correct to me to say "if you want to add/modify any Gatsby documentation, go to..."
It's just shorter and I think it's already implied that it's written with markdown and exists on a website. Let me know if there are reasons to keep the other details in!